propolis/src/chardev/sock.rs

```diff
@@ -137,6 +137,10 @@ impl UDSock {
                 if e.kind() != ErrorKind::AddrInUse {
                     return Err(e);
                 }
+                eprintln!(
+                    "Couldn't connect to {}, removing and trying again...",
```

Comment:
Considering it's a `bind(3)` call, not `connect(3)`, this verbiage is a little off